bls-signatures-ffi/src/lib.rs

```diff
@@ -55,20 +64,25 @@ pub unsafe extern "C" fn hash(
 ///
 /// * `flattened_signatures_ptr` - pointer to a byte array containing signatures
 /// * `flattened_signatures_len` - length of the byte array (multiple of SIGNATURE_BYTES)
+///
+/// Returns `NULL` on error. Result must be freed using `destroy_aggregate_response`.
```

Comment:
Observation: Seems a little weird to not use the `err_code` and `err_message` convention (in use by `rust-fil-proofs-ffi` and `rust-fil-sector-builder`). The caller isn't going to be provided with any information telling them why their `aggregate` call failed.

Reply:
We haven't implemented any of that convention in here, and I wanted sth to get in quickly. I agree using that would be an improvement.